src/functionalTest/resources/application.yaml

```diff
@@ -0,0 +1,53 @@
+management:
+  endpoint:
+    health:
+      show-details: "always"
+  endpoints:
+    web:
+      base-path: "/"
+      exposure:
+        include: health, info, prometheus
+  security:
+    enabled: false
+
+spring:
+  application:
+    name: Bulk Scan Payment Processor
```

Comment:
We could name it `Bulk Scan Payment Processor Functional Tests`, for example

Reply:
Will do